pkg/destroy/aws/aws.go

```diff
@@ -1206,7 +1206,7 @@ func r53ZonesToAWSObjects(zones []*route53.HostedZone, r53Client *route53.Route5
 			})
 			if err != nil {
 				if request.IsErrorThrottle(err) {
-					logger.Debugf("sleeping before trying to resolve tags for zone %s: %v", zone.Id, err)
+					logger.Debugf("sleeping before trying to resolve tags for zone %s: %v", *zone.Id, err)
```

Comment:
Is `zone.Id` guaranteed not to be nil?

Reply:
I think so.  We already deference it without checking on line 1225.  We could use `aws.StringValue(zone.Id)` though.

Reply:
> Is `zone.Id` guaranteed not to be nil?

If `zone.Id` is not `nil`, we have a big problem and want to panic ;).